tests/core/ckeditor/manual/assetscachekeycdn.html

```diff
@@ -0,0 +1,114 @@
+<div id="editor"></div>
+<button disabled id="loadButton">Load external editor</button>
+<div id="loading_box" style="margin: 20px 0"></div>
+<div id="editor_cdn">
+	<p>A simple editor</p>
+</div>
+<div id="fake_observable_item"></div>
+
+<script>
+	( function () {
+		if ( bender.tools.env.mobile || ( CKEDITOR.env.ie && CKEDITOR.env.version < 11 ) ) {
+			return bender.ignore();
+		}
+
+		// 1. Load local editor by default with bender.
+		// 2. Remove existing instance and add script with editor from CDN.
+		// 3. Wait for the new editor to be loaded from CDN and initialize it.
+		// 4. Add icons in IE <11 from local.
```

Comment:
Why are we ignore this test on IE < 11? And if so, maybe the point `4.` in comments and its corresponding logic is not needed?

Reply:
We ignore it because older IEs don't have sensible network debugging, so the test there does not make any sense.

And probably you're right about the logic for icons.